src/components/pages/pricing/plans/table/table.jsx

```diff
@@ -295,6 +336,12 @@ const Table = () => {
                   theme={isHighlightedColumn ? 'primary' : 'gray-15'}
                   to={labelList[key].buttonUrl}
                   tag_name={`Details Table Bottom > ${labelList[key].label}`}
+                  onClick={() => {
+                    const event = labelList[key].analyticsEvent;
+                    if (event.analyticsEvent) {
```

Comment:
I think you have an extra property access here

Reply:
Oh, thank you! Website is written in Javascript, with all disadvantages...

Fixed!